packages/api-sdk/src/medical/models/__tests__/demographics.test.ts

```diff
@@ -0,0 +1,45 @@
+import { contactSchema, emailSchema } from "../demographics";
+import { makeContact } from "./contact";
+
+describe("demographicsSchema", () => {
+  describe("contactSchema", () => {
+    it("accepts simple email", async () => {
+      const contact = makeContact({ email: "test@metriport.com" });
+      expect(() => contactSchema.parse(contact)).not.toThrow();
+    });
+
+    it("accepts null email", async () => {
+      const contact = makeContact();
+      contact.email = null;
+      expect(() => contactSchema.parse(contact)).not.toThrow();
+    });
+
+    it("accepts undefined email", async () => {
+      const contact = makeContact();
+      contact.email = null;
+      expect(() => contactSchema.parse(contact)).not.toThrow();
+    });
+  });
+
+  describe("emailSchema", () => {
+    it("accepts simple email", async () => {
+      expect(() => emailSchema.parse("test@metriport.com")).not.toThrow();
+    });
+
+    it("fails when gets null", async () => {
+      expect(() => emailSchema.parse(null)).toThrow();
+    });
+
+    it("fails when gets undefined", async () => {
+      expect(() => emailSchema.parse(undefined)).toThrow();
+    });
+
+    it("accepts email with dot", async () => {
+      expect(() => emailSchema.parse("test.two@metriport.com")).not.toThrow();
+    });
+
+    it("fails when gets email with colon", async () => {
+      expect(() => emailSchema.parse("test:two@metriport.com")).toThrow();
+    });
+  });
```

Comment:
we don't wanna check more cases? email without `@`? email with subdomain, etc?

Reply:
yeah, we can def add more cases, but I was not trying to replicate Zod's test, mostly trying to replicate scenarios we've been through and want to know what to expect - and have the code replicating that.

I'll add a few more

Reply:
done

Reply:
🚀 🚀